doc/onboarding.md

```diff
@@ -86,6 +86,13 @@ onboarding session.
       `semver-major` label
     * When adding a `semver-*` label, add a comment explaining why you're adding
       it. Do it right away so you don't forget!
+  * Please add the `author-ready` label for PRs where:
```

Comment:
All this list seems confusing grammar-wise. "where:" prefix does not combine smoothly with points like "that have no outstanding review comments and". It seems we have to unify the grammatical pattern of the points or reword the introductory sentence.

Reply:
Yeah, the list is a bit confusing. `the CI has been started` and `pending the CI outcome` seems redundant, for example. Or maybe there's a nuance there that I'm missing?

Reply:
About `the CI has been started` and `pending the CI outcome`: It should be clear that a) the CI has to be started and b) it does not have to be done when adding the label. Do you maybe have a better wording for that?

Reply:
I think `CI has been started` is sufficient. But if you want to be explicit, maybe `CI has been started (not necessarily finished)` or something like that?

Reply:
Addressed.